test/setup/appengine.tf

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-resource "google_app_engine_application" "app" {
-  project     = module.project.project_id
-  location_id = "us-central"
-}
+# resource "google_app_engine_application" "app" {
```

Comment:
nit: lets remove commented code

Reply:
I'll just remove the file since there's nothing else in it.